buildSrc/src/main/kotlin/splunk.java-conventions.gradle.kts

```diff
@@ -13,10 +13,10 @@ repositories {
   }
 }
 
-val otelVersion = "1.4.0"
-val otelAlphaVersion = "1.4.0-alpha"
-val otelInstrumentationVersion = "1.5.0-SNAPSHOT"
-val otelInstrumentationAlphaVersion = "1.5.0-alpha-SNAPSHOT"
+val otelVersion = "1.4.1"
+val otelAlphaVersion = "1.4.1-alpha"
+val otelInstrumentationVersion = "1.4.0"
+val otelInstrumentationAlphaVersion = "1.4.0-alpha"
```

Comment:
I've reverted to 1.4.0 for now -- we can't release a stable release based on a snapshot. Will revert it back in post-release PR.